js/ui/scheduler/recurrence_editor.js

```diff
@@ -431,7 +431,7 @@ class RecurrenceEditor extends Editor {
     _renderEditors($container) {
         this._recurrenceForm = this._createComponent($container, Form, {
             items: this._editors,
-            showValidationSummary: true,
+            showValidationSummary: false,
```

Comment:
Do we need render validation summary in Scheduler by default?

Reply:
In dxForm, there will be a div with fixed size that reserves empty space when validationSUmmary is enabled.
Recurrence form doesn't declare validation rules and should not reserve that space for summary.